### DISCUSSION TITLE
Essential oils

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#313] [General]
Do you use essential oils for a stress relief? Which ones are your favorite?

**Comment by Ozankey** [comment]
I used lavender for sleep and love it. Apart from that, I also tried peppermint and lemon grass but lavender is still the best.

**Comment by hernessteve** [comment]
Lavender and lemongrass oils work the best for me. I received a whole set of different oils and a new diffuser and it was my best present for the last couple of years. Never before i heard about [https://volantaroma.se/](https://volantaroma.se/) but i my mind is blown with their products.
